eternalcore-core/src/main/java/com/eternalcode/core/feature/essentials/container/LoomCommand.java

```diff
@@ -0,0 +1,52 @@
+package com.eternalcode.core.feature.essentials.container;
+
+import com.eternalcode.annotations.scan.command.DescriptionDocs;
+import com.eternalcode.containers.AdditionalContainerPaper;
+import com.eternalcode.containers.AdditionalContainerType;
+import com.eternalcode.core.notification.NoticeService;
+import dev.rollczi.litecommands.argument.Arg;
+import dev.rollczi.litecommands.command.execute.Execute;
+import dev.rollczi.litecommands.command.permission.Permission;
+import dev.rollczi.litecommands.command.route.Route;
+import org.bukkit.entity.Player;
+
+@Route(name = "loom", aliases = "lom")
```

Comment:
```suggestion
@Route(name = "loom")
```

Trochę sus then alias